tomviz/pybind11/CMakeLists.txt

```diff
@@ -1,10 +1,18 @@
+find_package(TBB)
+
 set(CMAKE_MODULE_LINKER_FLAGS "")
 pybind11_add_module(_wrapping
   OperatorPythonWrapper.cxx
   PipelineStateManager.cxx
+  ctvlib.cxx
+  eigenConversion.h
```

Comment:
I would like to see this code in a separate library. Can we create a subdirectory `ctvlib` and move the code there and create a new library `ctvlib`?

Reply:
@cjh1 yes, I can make a separate library and move the code into a separate folder inside of wrapping or in a different folder?

Reply:
I was thinking in the pybind11 directory. So from the top of the repo it would be `tomviz/pybind11/ctvlib`?

Reply:
@cjh1 hi, I've just moved the source code into its own subdirectory. Is this what you had in mind?